src/main/java/com/google/api/codegen/config/DiscoGapicInterfaceContext.java

```diff
@@ -268,6 +273,22 @@ public DiscoGapicMethodContext asRequestMethodContext(MethodModel method) {
         getFeatureConfig());
   }
 
+  @Override
+  public DiscoGapicMethodContext asNonLroMethodContext(
```

Comment:
So an RPC can be an LRO method, but there are contexts in which we don't want to treat it as an LRO method. For example, treating a method as an LRO method means it has a different method name (ends in `Async` for Java methods) and that it has a different return type, etc. 

In discogapic, because LRO is configurable, we need to preserve the non-LRO version of a generated method so as to not break the non-LRO method after it gets configured to be an LRO method.

Reply:
Do we want to be able to do this? There are no discogapic GA clients yet, so we can break them now (if this is for backward compatibility). Why would we want to be able to configure an lro method as non-lro method? Can generator detect that the method is LRO and fail in case if the corresponding LRO configuraiton is not provided?

Reply:
By default, without configuration, all methods are non-LRO. In the near future, RPCs will have to explicitly be configured to be LRO methods in the GAPIC config.

If we update the discovery document and start generating a new method, it will be non-LRO. If we later configure the method to be LRO, then it will break the previous non-LRO method because it will replace that non-LRO method. 

I propose that instead of an RPC being only LRO or non-LRO, that an LRO RPC is generated as both a pollable-Future LRO method and also as a regular non-LRO method.